src/AppInstallerCLIPackage/Package.appxmanifest

```diff
@@ -37,6 +37,16 @@
             <uap5:ExecutionAlias Alias="WinGetDev.exe" />
           </uap5:AppExecutionAlias>
         </uap5:Extension>
+      </Extensions>
+    </Application>
+    <Application Id="WinGetComServer" Executable="WinGetServer\WindowsPackageManagerServer.exe" EntryPoint="Windows.FullTrustApplication">

```

Comment:
><Application Id="WinGetComServer" Executable="WinGetServer\WindowsPackageManagerServer.exe" EntryPoint="Windows.FullTrustApplication"> [](http://example.com/codeflow?start=4&length=134)

Does this require changes in AppInstaller manifest or this is just for dev testing